packages/generator-single-spa/src/svelte/templates/svelte.package.json

```diff
@@ -1,33 +1,33 @@
 {
   "name": "<%= name %>",
+  "type": "module",
   "scripts": {
     "build": "concurrently <%- packageManager %>:build:*",
     "build:rollup": "rollup -c",
     "start": "rollup -c -w",
     "serve": "sirv dist -c",
-    "test": "jest",
+    "test": "vitest",
```

Comment:
Did the jest configuration no longer work after the upgrade to svelte? Why switch from jest to vitest?

Reply:
I'm ok switching to vitest for this generator. Jest typescript esm with node flags for vm modules is a pain.